# trajectory_folder_in: "/home/vit/git/reshaping_experiments/tmux_scripts/one_drone_test/trajectories"
trajectory_folder_in: "/home/vit/git/mvl_formation_control_core/ros_packages/mvl_formation_control/trajectories/six_uavs_static"
# trajectory_folder_in: "/home/vit/transformed_trajectories3"
# trajectory_folder_in: "/home/vit/git/mvl_formation_control_core/ros_packages/mvl_formation_control/trajectories/three_uavs_moving"
trajectory_folder_out: "/home/vit/git/reshaping_experiments/tmux_scripts/one_drone_gps/trajectories/six_uavs_static"
# trajectory_folder_out: "/home/vit/transformed_trajectories"

process_all: true # if true, all files in the trajectory_folder_in are processed

trajectory_files: ["trajectory1.txt", "trajectory2.txt", "circle_10_5.txt", "eight_8ms.txt"]

trajectory_dt: 0.2 # (s)

transformation:
  translation: 
    x: -10.0
    y: 5.0 
    z: 0.0
  rotation:
    heading: -0.7
    origin:
      x: 0.0
      y: 0.0
